### ISSUE TITLE
add flow-types 

### ISSUE THREAD

**Issue by FilipMessa** [thomaslindstrom/empty-string#1]
add flow types to package

**Comment by thomaslindstrom** [comment]
good idea. it's important to be strict about types. would you like to do a pr?

**Comment by ghost** [comment]
> 
> 
> add flow types to package



> 
> 
> good idea. it's important to be strict about types. would you like to do a pr?

it's important to be strict about types. I just submitted a [pr](https://github.com/thomaslindstrom/empty-string/pull/3).
